core-developers/responsibilities.rst

```diff
@@ -68,7 +68,7 @@ the ability to license your code means it can be put under the PSF license so
 it can be legally distributed with Python.
 
 This is a very important step! Hopefully you have already submitted a
-contributor agreement if you have been submitting patches. But if you have not
+contributor agreement if you have been submitting pull requests. But if you have not
```

Comment:
You have to sign the CLA in order to become a contributor, right?

Reply:
You have to sign it to have PRs _merged_, but you can technically still submit PRs before that.